### ISSUE TITLE
Display Chattopic as name

### ISSUE THREAD

**Issue by Vampire** [EionRobb/skype4pidgin#632]
The name of chats in Skype are displayed in Pidgin as topic.
It would be nice if the Topic would also be used as name in Pidgin.
This is, if a chat you have not added to Pidgin opens because someone created it or wrote a message.
Currently the Skype-internal cryptic name is shown in the title bar.

Also when adding a chat, you search it in the roomlist and select add, but then the Alias field is empty, would be nice if it would be pre-filled with the Topic.

**Comment by EionRobb** [comment]
Unfortunately, both the "cryptic name as title" and "alias field not pre-populated" are limitations of libpurple that a protocol plugin can't provide

The only way around this is for the plugin to automatically and forcibly add the group chat to the buddy list, which you wouldn't be able to delete. I guess I could add it as an account option?

**Comment by Vampire** [comment]
Yes, sounds not too bad if it is optional, especially as the chats are also forcibly present in the Skype client.
Would it be possible to change the alias then?
And would you be able to move the chats into different groups, so that one could collapse the chats one is not usually interested in?
Maybe also the Group into which they get added by default could be configurable.
